### DISCUSSION TITLE
ISO metadata for OGC API Feature Services

### DISCUSSION THREAD

**Discussion by Kate-Lyndegaard** [INSPIRE-MIF/helpdesk-validator#1246] [Q&A]
Hi,

What values does the INSPIRE validator expect in `<SV_ServiceIdentification>` for ISO metadata describing OGC API Feature services?

For example, what does the INSPIRE validator expect in the `<containsOperations>` element? What is the name of the operation? What should be added as linkage URL?

Also, what is the expected character string for `<serviceTypeVersion>`? Are there any guidelines?

**Comment by alitka** [comment]
In Germany, we worked out the following conventions: 
* The link to the landing page, which partially describes the service and contains further links, must be provided.
* In addition, the <code>protocol</code> element containing the service name according to the INSPIRE Registry code list ‘INSPIRE Protocol values’, here: ‘OGC API features’. The specification as gmx:Anchor is recommended. Alternatively, the characterString data type can be used.
* The function of the stored URL in CI_OnlineResource can be annotated in the <code>function</code> element using the terms of the code list B.5.3 CI_OnLineFunctionCode from [ISO 19115]. This is recommended by [INSTG MD], 4.1.3.1 (Recommendation 3.4) and adopted in the GDI-DE, but is not mandatory. An obvious and sensible use in metadata for services in general is seen for the following term in the code list: _information_ - for the link to the capabilities document of the service, the landing page at OGC-API or a website that contains further instructions or information

**example**
```<srv:serviceType>```
```<gco:LocalName codeSpace="http://inspire.ec.europa.eu/metadata-codelist/SpatialDataServiceType">download</gco:LocalName>```
```</srv:serviceType>```
```<srv:serviceTypeVersion>```
```<gmx:Anchor xlink:href="http://www.opengis.net/def/interface/ogcapi-features">OGC API-Features</gmx:Anchor>```
```</srv:serviceTypeVersion>```
...
```<gmd:transferOptions>```
```<gmd:MD_DigitalTransferOptions>```
```<gmd:onLine>```
```<gmd:CI_OnlineResource>```
```<gmd:linkage>```
```<gmd:URL>https://ogc-api.geobasis-bb.de/basemap-bebb</gmd:URL>```
```</gmd:linkage>```
```<gmd:protocol>```
```<gmx:Anchor xlink:href="http://www.opengis.net/def/interface/ogcapi-features">OGC API-Features</gmx:Anchor>```
```</gmd:protocol>```
```<gmd:function>```
```<gmd:CI_OnLineFunctionCode codeList="https://standards.iso.org/iso/19139/resources/gmxCodelists.xml#CI_OnLineFunctionCode" codeListValue="information"/>```
```</gmd:function>```
```</gmd:CI_OnlineResource>```
```</gmd:onLine>```
```</gmd:MD_DigitalTransferOptions>```
```</gmd:transferOptions>```
